smart_campus/app/templates/app/index.html

```diff
@@ -26,29 +26,29 @@
               <img class="ui small image" src="{% static 'app/icon/reward.jpg' %}">
             </div>
             <div class="ui bottom" style="color:#008080; text-decoration:none;">
-              管理收藏品
+              收藏品
             </div>
           </a>
         </div>
         <div class="column">
-          <a class="ui raised link centered card" id="marginCard" href="{% url 'Question List Page' %}">
+          <a class="ui raised link centered card" id="marginCard" href="{% url 'Beacon List Page' %}">
             <div class="image">
-              <img class="ui small image" src="{% static 'app/icon/question.jpg' %}">
+              <img class="ui small image" src="{% static 'app/icon/beacon.jpg' %}">
             </div>
             <div class="ui bottom" style="color:#008080; text-decoration:none;">
-              管理挑戰
+              B eacon
```

Comment:
'B eacon' -> 'Beacon'

Reply:
Fixed it.
Btw, should I merge the current develop branch or wait for the MEDIA_URL problem solved?

Reply:
I'll merge this branch now so that you can create another branch for fixing that problem.